app/scripts/routes.js

```diff
@@ -950,6 +950,9 @@
             .when('/editadhocquery/:id', {
                 templateUrl: 'views/adhocquery/editadhocquery.html'
             })
+            .otherwise({
+                templateUrl: "views/404.html"
```

Comment:
Let's create a separate folder for `errors` inside `views` and move the file there